crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -841,7 +841,9 @@ impl CheckpointAggregator {
                     next_index: 0,
                     digest: summary.digest(),
                     summary,
-                    signatures: StakeAggregator::new(self.epoch_store.committee().clone()),
+                    signatures: StakeAggregator::new(Arc::new(
```

Comment:
Maybe not for this PR, but we should probably change PerEpochStore::committee to be an arc, so that instead of making new arc every time and cloning entire committee, we just keep an Arc in epoch store and clone the Arc